src/olympia/api/tests/test_fields.py

```diff
@@ -193,6 +193,22 @@ def test_field_null(self):
         result = field.to_representation(field.get_attribute(self.addon))
         assert result is None
 
+    def test_field_value_null(self):
+        self.addon = addon_factory(slug='lol', name=None, description=None)
+
+        request = Request(self.factory.get('/', {'lang': 'en-US'}))
+        mock_serializer = serializers.Serializer(context={'request': request})
+        field = self.field_class()
+
+        # field.bind('name', mock_serializer)
+        # result = field.to_representation(field.get_attribute(self.addon))
+        # assert result is None
```

Comment:
commented out code, I think that should be tested too?

Reply:
Yeah, I was just experimenting with another part of the test. I'll uncomment, it should still pass.